src/cli/commands/init.js

```diff
@@ -89,12 +91,17 @@ export async function run(config: Config, reporter: Reporter, flags: Object, arg
       question: 'license',
       default: String(config.getOption('init-license')),
     },
+    {
+      key: 'private',
+      question: 'private',
+      default: '',
```

Comment:
I think `1` i shorter way user want to fill this key, so i use `yn`. And everything can be become a boolean when we use `yn`. So, if users want to change it, they can change it later by edit `package.json` file. Just `boolean` value is approved.